packages/core/src/components/ProgressBar/index.tsx

```diff
@@ -4,29 +4,62 @@ import useStyles from '../../hooks/useStyles';
 import { styleSheet } from './styles';
 
 export type Props = {
+  /** Dangerous/failure status (red). */
+  danger?: boolean;
```

Comment:
Should probably add those prop types that disable all these from being used in unison.

Reply:
done!